src/core/border_router/routing_manager.cpp

```diff
@@ -754,6 +754,7 @@ void RoutingManager::SendRouterAdvertisement(RouterAdvTxMode aRaTxMode)
         destAddress.SetToLinkLocalAllNodesMulticast();
 
         error = mInfraIf.Send(raMsg.GetAsPacket(), destAddress);
+        Get<Ip6::Ip6>().GetBorderRoutingCounters().mRaTx++;
```

Comment:
Do we want to count all attempts? Or just successful ones (i.e. when error == kErrorNone)?

Reply:
I'm inclined to count all attempts assuming that the failures should be rare. If errors aren't negligible, maybe I should count successful transmissions and failed transmissions respectively.

Reply:
We have seen send failures occur in the past. I think it would be useful to count success and failure separately.